third_party/gpus/cuda/BUILD.tpl

```diff
@@ -58,10 +58,8 @@ cc_library(
     linkopts = [
         "-ldl",
         "-lpthread",
-    ] + select({
-        "@//tensorflow:darwin": [],
```

Comment:
This does exist, but it's just a typo, right?
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/BUILD#L41

Probably should be "//tensorflow:darwin" without the @